src/SideBoardCard.jsx

```diff
@@ -1,22 +1,32 @@
 import React from "react";
 import "./styles.css";
 
-export default function SideBoardCard({ value, id, total, setTotal, isVisible, setIsVisible }) {
-    // Allows sideboard cards to affect total
-    // REMOVE SIDE CARD FROM BOARD ONCE USED
-    
-    function useSideCard() {
-      if (total + value < 0) {
-        console.log("Cannot bring total below zero!");
-      } else {
-        setTotal(total + value);
-        setIsVisible(!isVisible);
-      }
+export default function SideBoardCard({ card, total, setTotal, setSideBoard }) {
+  // Allows sideboard cards to affect total
+  // REMOVE SIDE CARD FROM BOARD ONCE USED
+
+  const { id, value, isVisible } = card;
+
+  function useSideCard() {
+    if (total + value < 0) {
+      console.log("Cannot bring total below zero!");
+    } else {
+      setTotal(total + value);
+      setSideBoard((prevDeck) => {
+        const cardsDup = JSON.parse(JSON.stringify(prevDeck)); // little trick to duplicate array
+        const updateCard = cardsDup.find((c) => c.id === id);
```

Comment:
Find the card we just clicked within the deck of cards, now we can manipulate the `isVisible` state on that one card.